litestar_sphinx_theme/__init__.py

```diff
@@ -39,9 +39,10 @@ def update_html_context(
         "twitter_link",
         "https://twitter.com/LitestarAPI",
     )
-    # waiting for shibuya#32
-    # context["reddit_link"] = theme_options.get(
-    #     "reddit_link",
+    context["reddit_url"] = theme_options.get(
```

Comment:
Shibuya theme supports Reddit natively now.

Reply:
This is so that we dont need to define it in each of our projects conf.py 😄